var/spack/repos/builtin/packages/intel-parallel-studio/package.py

```diff
@@ -347,3 +354,17 @@ def setup_environment(self, spack_env, run_env):
             run_env.prepend_path('MIC_LD_LIBRARY_PATH',
                                  join_path(self.prefix, 'ipp', 'lib', 'mic'))
             run_env.set('IPPROOT', join_path(self.prefix, 'ipp'))
+
+    def setup_dependent_package(self, module, dep_spec):
+        if (self.spec.satisfies('+all') or self.spec.satisfies('+mpi')):
+            # Check for presence of bin64 or bin directory
+            if os.path.isdir(self.prefix.bin):
+                bindir = self.prefix.bin
+            elif os.path.isdir(join_path(self.prefix, 'bin64')):
+                bindir = join_path(self.prefix, 'bin64')
+            else:
+                raise "No suitable bindir found"
```

Comment:
in another PR, @adamjstewart said that this should probably be `raise RuntimeError('No suitable bindir found')`